ocs_ci/templates/CSI/addons/NamespaceSkipReclaim.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
nitpick - Should the name of the file have "Namespace" in it? 
maybe simply "Reclaimspace_skip" to align with existing filenames?

Reply:
+1

Reply:
Done, thank you!